pkg/config/load_test.go

```diff
@@ -85,6 +85,53 @@ func TestExtractRepoElementsFromPath(t *testing.T) {
 	}
 }
 
+func TestInfo_IsComplete(t *testing.T) {
+	testCases := []struct {
+		name        string
+		info        Info
+		expectError string
```

Comment:
nit: a little simpler would have just been `expectError error` and use `errors.New()` but OK